en/micro-integrator/docs/setup/deployment/k8s-pipeline/adding-test-cases.md

```diff
@@ -0,0 +1,107 @@
+# Adding test cases
+
+## Add your own test case to the Kubernetes Pipeline
+
+This document assumes that you have completed the instructions in the
+following pages.
+
+1.  [Quick Start Guide](/setup/deployment/k8s-pipeline/quick-start-guide/)
+
+2.  [Testing The Pipeline
+    Environment](/setup/deployment/k8s-pipeline/testing-the-pipeline-environment/)
+
+Test cases are available in the chart source repository under a
+directory named `tests`. This guide demonstrates how you can add test
+cases.
+
+1.  Create a fork of the [sample chart source
+    repository](https://github.com/wso2-incubator/cicd-sample-chart-mi).
+
+2.  Create a clone of the forked chart source repository using the
+    following command.
+    
+    ``` xml
+    $ git clone https://github.com/<ORGANIZATION_NAME>/cicd-sample-chart-mi.git
```

Comment:
I'm wondering if org is the right term here. If it's an individual developer then it would be <GitHub_Username> or something similar.

Reply:
+1. Is it possible to mention both?

Reply:
@samgnaniah: Resolved this as reflected in the following https://github.com/wso2/docs-apim